### PR TITLE
Updates for data loader xml comments.

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

File: src/GraphQL/DataLoader/DataLoaderBase.cs [hunk 1]
```diff
@@ -19,7 +19,15 @@ public abstract partial class DataLoaderBase<TKey, T> : IDataLoader, IDataLoader
         private DataLoaderList _list;
         private readonly Dictionary<TKey, DataLoaderPair<TKey, T>> _cachedList;
         private readonly object _sync = new object();
+
+        /// <summary>
+        /// Returns the maximum number of keys passed to the fetch function at a time
```
Comment (sungam3r) [inline src/GraphQL/DataLoader/DataLoaderBase.cs:24]:
```suggestion
        /// Returns the maximum number of keys passed to the fetch function at a time.
```

File: src/GraphQL/DataLoader/DataLoaderBase.cs [hunk 2]
```diff
@@ -19,7 +19,15 @@ public abstract partial class DataLoaderBase<TKey, T> : IDataLoader, IDataLoader
...
+        /// </summary>
         protected internal readonly int MaxBatchSize;
+
+        /// <summary>
+        /// Returns the equality comparer to be used, or null for the default equality comparer
```
Comment (sungam3r) [inline src/GraphQL/DataLoader/DataLoaderBase.cs:29]:
```suggestion
        /// Returns the equality comparer to be used, or null for the default equality comparer.
```

File: src/GraphQL/DataLoader/DataLoaderContextExtensions.cs [hunk 3]
```diff
@@ -11,8 +11,19 @@ namespace GraphQL.DataLoader
     /// </summary>
     public static class DataLoaderContextExtensions
     {
+        /// <summary>
+        /// Returns a delegate which calls the delegate passed this method, stripping off the <see cref="CancellationToken"/> in the process.
```
Comment (sungam3r) [inline src/GraphQL/DataLoader/DataLoaderContextExtensions.cs:15]:
```suggestion
        /// Returns a delegate which calls the delegate passed to this method, stripping off the <see cref="CancellationToken"/> in the process.
```

File: src/GraphQL/DataLoader/DataLoaderContextExtensions.cs [hunk 4]
```diff
@@ -11,8 +11,19 @@ namespace GraphQL.DataLoader
...
+        /// </summary>
+        /// <typeparam name="TResult">The type of the return value of the delegate.</typeparam>
+        /// <param name="func">The delegate to call.</param>
         public static Func<CancellationToken, TResult> WrapNonCancellableFunc<TResult>(Func<TResult> func) => cancellationToken => func();
 
+        /// <summary>
+        /// Returns a delegate which calls the delegate passed this method, stripping off the <see cref="CancellationToken"/> in the process.
```
Comment (sungam3r) [inline src/GraphQL/DataLoader/DataLoaderContextExtensions.cs:22]:
```suggestion
        /// Returns a delegate which calls the delegate passed to this method, stripping off the <see cref="CancellationToken"/> in the process.
```

**Comment by Shane32** [comment]
If you can review this tonight @sungam3r then I can spend some time tonight on a new PR targeting `develop` to clean up some issues I found while adding these comments.

**Review by sungam3r — APPROVED** [review]

**Comment by Shane32** [comment]
Sorry my comment addressed the wrong PR -- lol !!!

**Pull request merged**